src/tests/ftest/util/dfuse_utils.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, namespace, command):
         self.sys_name = FormattedParameter("--sys-name {}")
         self.singlethreaded = FormattedParameter("--singlethread", False)
         self.foreground = FormattedParameter("--foreground", False)
+        self.enable_caching = FormattedParameter("--enable-caching", False)
+        self.enable_wb_caching = FormattedParameter("--enable-wb-cache",
```

Comment:
We use 100 characters line-length now and I think this would probably fit in one line.

Reply:
Okay got it. I should probably then update the disable_wb_caching then too. I also noticed for enable|disable_wb_caching is used in this file versus the tense enable|disable_wb_cache. I'll fix that up now too.